### ISSUE TITLE
Action editor right panel closes with any change

### ISSUE THREAD

**Issue by tylergarcia511** [FlutterFlow/flutterflow-issues#2813]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

After opening the action editor and attempting to make any change to any action, the right panel in which you make adjustments/changes closes out. Selecting an action such as "Navigate" closes the panel, forcing me to re-select it to choose the navigation page; and, upon choosing the page, closes again.

### Expected Behavior

Able to modify actions without having to continuously re-select the action

### Steps to Reproduce

1. Open action editor
2. Add action
3. Select any action on the right hand side, and watch the action editor close as soon as a change is made

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4WiMmBw891ofpE15DucMdVoCMvMjsmU40visN+Sg8cIZDnE4wPXPykP1doTcOiTFZYMFv9in8ezMLSv9r9DeQHAzeeGK5q/rl5EjyVcH26aa6SCoeCOn99P8pTJEuV0KSNsyZSVctfcUYWwUyYBe+WVhTcGOO0DEQWRsn/UaPPpA+QAj+6Xn0Jg1ZSfCvr

### Context

I can't make changes to actions..

### Visual documentation

https://www.loom.com/share/17df87407f56434890abef054278a78f

### Environment

```markdown
- FlutterFlow version: 4.1.45
- Platform: Mac
- Browser name and version: Safari but using desktop app
- Operating system and version affected: latest macOS
```

### General

- [ ] I was collaborating with other users in the same project prior to data loss
- [ ] I navigated between different project views leading up to data loss
- [ ] The UI was running very slowly prior to data loss
- [ ] A crash or other bug occurred prior to data loss
- [ ] Data was not saved in a snapshot
- [ ] I was using flutterflow from multiple devices
- [ ] I was using flutterflow from multiple platforms (e.g. both web and MacOS Desktop app)

### Relative to the time the changes were made, data was lost within

- [ ] 1 minute
- [ ] 1 hour
- [ ] 1 day
- [ ] 1 week

### When following my steps to reproduce, data loss happens

- [ ] ~100% of the time
- [ ] ~50% of the time
- [ ] ~10% of the time
- [ ] ~1% of the time
- [ ] Very rarely or only encountered once

**Comment by tylergarcia511** [comment]
For more context, this started immediately after I updated today. I'm not sure if there is a way to downgrade so I can at least get around this for now? Thanks!

**Comment by rzambroni** [comment]
Hey @tylergarcia511, according to our team this was already fixed, could you please refresh the editor for the latest updates and verify if this is still happening on your end? Thanks!

**Comment by tylergarcia511** [comment]
After force quitting and reopening, the application says I am on the latest version after it checks for updates. It still is happening.

**Comment by sabatage3** [comment]
I have the same issue, it started happening after the last update, so it's a bug, also, when you create a variable (from a firestore query for example) if you try to write the variable name, the action closes after inserting the first character, so it's impossible to use at the moment, and yeah, it's not fixed yet

**Comment by tylergarcia511** [comment]
Yeah, I've just been running off an old version Dmg I have and it isn't happening.

**Comment by stevey6404** [comment]
Still occurring via Mac app...

**Comment by ambience-kim** [comment]
I too have this problem and find it very difficult to work.
This problem still occurs and I'm using the latest version of the Mac OS FF app.

Need to find an older version of dmg?
This problem is still happening, why is the issue closed?

**Comment by paulperez-dev** [comment]
Hi @tylergarcia511, @sabatage3, can you please verify if the issue persists?<br><br>@ambience-kim, can you please share the version you are using?

**Comment by ambience-kim** [comment]
> Hi @tylergarcia511, @sabatage3, can you please verify if the issue persists?@ambience-kim, can you please share the version you are using?

I'm using the FF App for Mac OS. (V 4.1.47)
I checked this issue again yesterday after checking the comments here, and yesterday this issue continued to occur.

In addition, since yesterday, an additional issue occurred where the FF App screen was refreshed when the Mac screen was switched, so I was unable to use the Mac OS app.
https://github.com/FlutterFlow/flutterflow-issues/issues/2855

So yesterday I did some FF development using a web browser (Chrome).
(This issue did not occur in the web version of FF.)

And after seeing your comment today, I tried to reproduce this issue using the App for Mac OS, but now this issue no longer occurs.
I didn't update the App for Mac OS that I'm using yesterday and today, but now I no longer have this issue.

Because the response speed of the FF web version is slower than the Mac OS app, I have not used it for almost two months, but after using the web version of FF yesterday, this issue does not currently occur in the Mac OS app.
